### ISSUE TITLE
Error while searching

### ISSUE THREAD

**Issue by Progluver** [josh0xA/darkdump#3]
`python3 darkdump.py --query testing`

Output: 

`Operating System: linux`

     ____          _     _               
    |    \ ___ ___| |_ _| |_ _ _____ ___ 
    |  |  | .'|  _| '_| . | | |     | . |
    |____/|__,|_| |_,_|___|___|_|_|_|  _|
                                    |_|  

        Developed By: Josh Schiavone
        https://github.com/josh0xA  
            joshschiavone.com     
            
Searching For: testing on page: 1...
```
Traceback (most recent call last):
  File "darkdump.py", line 169, in <module>
    darkdump_main()
  File "darkdump.py", line 165, in darkdump_main
    Darkdump(cfg.__darkdump_api__, query).crawl_api()
  File "darkdump.py", line 105, in crawl_api
    json_data = darksearch_url_response.json()
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 900, in json
    return complexjson.loads(self.text, **kwargs)
  File "/usr/lib/python3.7/json/__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.7/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.7/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

I get this error while running it on google compute engine. But it works fine in windows. What must be the reason for this error?

**Comment by xXLOLYOUXx** [comment]
I also get the same error. Any fix?

**Comment by josh0xA** [comment]
Darkdump is fixed, released as Darkdump 2.0. Have fun!
